firestore-bigquery-export/extension.yaml

```diff
@@ -192,10 +192,11 @@ params:
   - param: BIGQUERY_PROJECT_ID
     label: Project Id
     description: >-
-      Override the default project bigquery instance. 
+      Override the default project bigquery instance.
       This can allow updates to be directed to a bigquery instance on another project.
     type: string
-    required: false
+    default: ${param:PROJECT_ID}
```

Comment:
are you sure this will works?

Reply:
Confirming change updated to `default: ${PROJECT_ID}`